<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>581</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>575</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>youtube bluster</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="proxybutton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>160</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load proxy file</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>44</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Proxy list</string>
    </property>
   </widget>
   <widget class="QLabel" name="views_required">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Views required</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="views_required_input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>views done :</string>
    </property>
   </widget>
   <widget class="QLabel" name="view_count">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>490</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_browsing">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>490</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>start browsing</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>531</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="browser_layout"/>
   </widget>
   <widget class="QTextEdit" name="urlbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">label_3{
float:center;
}</string>
    </property>
    <property name="text">
     <string>   Browser window</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="video_length">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>video length in seconds</string>
    </property>
   </widget>
   <widget class="QListWidget" name="proxy_list">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>search url(where the video link can be found</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>video id(after v=)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="video_id">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>referer site</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="referer">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
